documentation/troubleshooting.md

```diff
@@ -371,6 +371,12 @@ inventory-config:
     shell-command: /bin/sh -c
 ```
 
+## I can't access sensitive output from a task using the `pcp` transport
+
+Sensitive output is not supported when running tasks using the `pcp` transport.
+For more information, see the [`bolt_shim`
+documentation](https://github.com/puppetlabs/puppetlabs-bolt_shim/blob/main/docs/sensitive_task_output.md).
```

Comment:
I feel silly for asking this, but are you writing the `sensitive_task_output.md` because I tried the link and it didn't work?

Reply:
@artlawson That file hasn't been merged in yet. Once it is, the link will work. :) Relevant PR is [here](https://github.com/puppetlabs/puppetlabs-bolt_shim/pull/25).

Reply:
😅😅